packages/web-components/fast-foundation/src/badge/badge.ts

```diff
@@ -39,21 +18,13 @@ export class Badge extends FoundationElement {
      */
     @attr({ mode: "boolean" })
     public circular: boolean;
-
-    public generateBadgeStyle = () => {
-        if (!this.fill && !this.color) {
-            return;
-        }
-
-        const fill: string = `background-color: var(--badge-fill-${this.fill});`;
-        const color: string = `color: var(--badge-color-${this.color});`;
-
-        if (this.fill && !this.color) {
-            return fill;
-        } else if (this.color && !this.fill) {
-            return color;
-        } else {
-            return `${color} ${fill}`;
-        }
-    };
+    private circularChanged(oldValue: boolean, newValue: boolean): void {
```

Comment:
I wonder if we could be more flexible here and allow badge implementations to have a broader range of options ie. circular, portrait, thumb, etc.... and let implementers customize more?  Just a thought.